pyscriptjs/src/utils.ts

```diff
@@ -109,3 +111,35 @@ export function getAttribute(el: Element, attr: string): string | null {
     }
     return null;
 }
+
+export function calculatePaths(fetch_cfg: FetchConfig[]) {
```

Comment:
why did yoy put this code into `utils.ts`?
It's specifically for `py-config`, it should go into `pyconfig.ts.`

Reply:
put it in `plugins/fetch.ts` as discussed on slack